hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1946,4 +1946,15 @@ possible configurations would overwhelm and obscure the important.
       responses with complete data.
     </description>
   </property>
+  <property>
+    <name>hbase.rpc.rows.size.threshold.reject</name>
+    <value>false</value>
+    <description>
+      If value is true, RegionServer will abort RPC multi() requests with number of rows
```

Comment:
Some improvements is needed here.
Client/Admin, if he/she is not a developer or like else, doesn't know what is multi() request. So, here we could be more clear and informative. (In fact, it's just a batch of Puts/Deletes)

Reply:
Oh, yes, I didn't realize this. Sure, let me update.
Thanks @Reidddddd

Reply:
Updating this to:
```
If value is true, RegionServer will abort batch requests of Put/Delete with number of rows
in a batch operation exceeding.......
```